src/components/questionary/questionaryComponents/FileUpload/FileUploadDefinition.tsx

```diff
@@ -25,7 +25,8 @@ export const fileUploadDefinition: QuestionaryComponentDefinition = {
     answerRenderer: FilesAnswerRenderer,
     questionRenderer: defaultRenderer.questionRenderer,
   },
-  createYupValidationSchema: () => Yup.array().of(Yup.string()),
+  createYupValidationSchema: () =>
+    Yup.array().of(Yup.object({ id: Yup.string().required() })),
```

Comment:
I think we added that file can also have a figure to the schema
`Yup.array().of(Yup.object({ id: Yup.string().required(), figure: Yup.string() }))`

Reply:
This is temporary just to make tests pass and the next PR will make this validation schema come from the validation package and there it is already set up. But thanks for pointing it.